### PR TITLE
feat: support a built-in parse

### PR DESCRIPTION
**Pull request by swain**
## Motivation
This was a great feature request from @danperkins. One less thing for new consumers to configure 🙌🏻.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,12 +142,6 @@ const router = new Router();
 
 implementSchema(Schema, {
   on: router,
-  parse: (ctx, { schema, data }) => {
-    // validate that `data` matches `schema`, using whatever
-    // library you like, and return the parsed response.
-
-    return data;
-  },
```
Comment (aecorredor) [inline README.md:150]: Should we still mention that this can be overridden?

File: src/integration.koa.test.ts [hunk 2]
```diff
@@ -256,3 +256,43 @@ test('introspection', async () => {
...
+
+      expect(result).toMatchObject({
+        status: 400,
+        data: "The request did not conform to the required schema: payload must have required property 'id'",
+      });
+    },
+  );
+});
+
+test('default parsing for GET', async () => {
```
Comment (aecorredor) [inline src/integration.koa.test.ts:279]: Are we missing a test for the possible override? Or is that already tested somewhere else and it wasn't surfaced in this diff?

File: README.md (hunk 1 above)
Comment (swain) [reply README.md:150]: Sure.

File: src/integration.koa.test.ts (hunk 2 above)
Comment (swain) [reply src/integration.koa.test.ts:279]: All of the existing tests validate custom parsers

**Review by aecorredor — APPROVED** [review]

**Review by xdumaine — APPROVED** [review]

**Pull request merged**